QgisModelBaker/gui/generate_project.py

```diff
@@ -190,10 +191,37 @@ def accepted(self):
         configuration.dbschema = configuration.dbschema or configuration.database
         self.save_configuration(configuration)
 
-        # create schema with superuser
         db_factory = self.db_simple_factory.create_factory(db_id)
-        res, message = db_factory.pre_generate_project(configuration)
 
+        try:
+            # raise warning when the schema or the database file already exists
+            config_manager = db_factory.get_db_command_config_manager(configuration)
+            db_connector = db_factory.get_db_connector(
+                config_manager.get_uri(configuration.db_use_super_login) or config_manager.get_uri(), configuration.dbschema)
+
+            if db_connector.db_or_schema_exists():
+                if interlis_mode:
+                    warning_box = QMessageBox(self)
+                    warning_box.setIcon(QMessageBox.Warning)
```

Comment:
Is the "warning" a read circle on your computer? I wonder if we should add this more as an information than a warning... It's just something to be aware of, not something that's bad per se.

Reply:
It's the exclamation mark sign you see above. We use the same when the xtf already exists on export. For me an information would be okay as well, but then I would make the "xtf already exists" an information as well. What do you think?

Reply:
I would say, the xtf *always* overwrites information. The import *could* overwrite information or just add more information to the existing data.

Reply:
Okay then.